server/pages/index.js

```diff
@@ -83,6 +83,22 @@ function hashFile( path ) {
 }
 
 /**
+ * Generate a URL for a file name with a hash
+ * @param {string} filename - the file name to hash
+ * @param {string} hash - a hash generated by hashFile
+ * @returns {string} a hashed URL
+ **/
+function getUrl( filename, hash ) {
+	return URL_BASE_PATH + '/' + filename + '?' + qs.stringify( {
+		v: hash
+	} );
+}
+
+function getHashedUrl( filename ) {
+	return getUrl( filename, hashFile( process.cwd() + SERVER_BASE_PATH + '/' + filename ) );
```

Comment:
CWD is maybe not the best option here, since it'll depend on when you run the server from...
maybe something like
`path.resolve( path.join( __dirname, '..', '..' ) )` ?

Reply:
CWD is what we use at present...

Reply:
But then we _must_ run the server from the root directory.
Not a huge issue, but might bite us someday...